src/script/components/InputBar/InputBar.tsx

```diff
@@ -898,8 +901,8 @@ const InputBar = ({
 
   return (
     <div
-      id="conversation-input-bar"
-      className={cx('conversation-input-bar', {'is-right-panel-open': isRightSidebarOpen})}
+      id={conversationInputBarClassName}
+      className={cx(`${conversationInputBarClassName}`, {'is-right-panel-open': isRightSidebarOpen})}
```

Comment:
I think it can be like that :)

```suggestion
      className={cx(conversationInputBarClassName, {'is-right-panel-open': isRightSidebarOpen})}
```

Reply:
👍